components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -213,7 +219,70 @@ const ConfigurationAnnouncementBar: React.FC<Props> = (props: Props) => {
             );
         }
 
-        if (isLicenseExpiring(props.license) && !props.dismissedExpiringLicense) {
+        const daysUntilLicenseExpires = daysToLicenseExpire(props.license);
+        if (isTrialLicense(props.license) && daysUntilLicenseExpires <= 14 && !props.dismissedExpiringTrialLicense) {
+            const purchaseLicense = (
+                <PurchaseLink
+                    buttonTextElement={
+                        <FormattedMarkdownMessage
+                            id='announcement_bar.error.purchase_a_license_now'
+                            defaultMessage='Purchase a License Now'
+                        />
+                    }
+                />
+            );
+
+            let message = (
+                <>
+                    <img
+                        className='advisor-icon'
+                        src={alertIcon}
+                    />
+                    <FormattedMarkdownMessage
+                        id='announcement_bar.error.trial_license_expiring'
+                        defaultMessage='**There are {days} days left on your free trial.**'
+                        values={{
+                            days: daysUntilLicenseExpires,
+                        }}
+                    />
+                </>
+            );
+
+            let announcementBarType = AnnouncementBarTypes.ANNOUNCEMENT;
+
+            if (daysUntilLicenseExpires <= 1) {
+                message = (
+                    <>
+                        <img
+                            className='advisor-icon'
+                            src={warningIcon}
+                        />
+                        <FormattedMarkdownMessage
+                            id='announcement_bar.error.trial_license_expiring_last_day'
+                            defaultMessage={'**This is the last day of your free trial. Purchase a license now to continue using E10 & E10 features.**'}
```

Comment:
```suggestion
                            defaultMessage={'**This is the last day of your free trial. Purchase a license now to continue using E10 & E20 features.**'}
```

Minor correction in text.

Reply:
Thank you!